url_dictionary.gemspec

```diff
@@ -1,20 +1,19 @@
-# coding: utf-8
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'url_dictionary'
 
 Gem::Specification.new do |spec|
   spec.name          = "url_dictionary"
   spec.version       = UrlDictionary::VERSION
-  spec.authors       = ["Niels Buus"]
-  spec.email         = ["nb@lokalebasen.dk"]
+  spec.authors       = ["Lev Lukomskyi"]
+  spec.email         = ["ll@matchoffice.com"]
```

Comment:
hehe, because I've rewritten all I thought maybe I can change author.. But yeah, maybe not. Maybe it would be better to just change it to "MatchOffice" to not bother with this data.